### DISCUSSION TITLE
🎦 Demo repository

### DISCUSSION THREAD

**Discussion by FH-Inway** [dev-cycles/contextive#69] [Ideas]
Along with the videos (see #68), it would be good to have a demo repository that people can use to get to know contextive interactively.

When looking at the other repos in the [dev-cycles](https://github.com/dev-cycles) org,  there are two that showcase how to use contextive across multiple repos. That's an advanced use case and the definitions.yml is a bit short. Not a good demo for newcomers, I think. 

In @chrissimon-au 's personal repos, there are [tdd-ddd-demo-dotnet](https://github.com/chrissimon-au/tdd-ddd-demo-dotnet) and [tdd-ddd-demo-java](https://github.com/chrissimon-au/tdd-ddd-demo-java), which also serve as a good demo for contextive. But they take some digging to find. 

It would be good to have a link in the readme of this repo to a demo repo in the dev-cycles org that people can use to try out contextive. Preferably with a .devcontainer and a codespaces link where everything is already set up. It could get pretty fancy with a [code tour](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour) (another nice documentation tool).

**Comment by chrissimon-au** [comment]
Great suggestion, and I really like the look of `code tour` - I'll take a look at setting up a demo repo.

I appreciate it's also a bit tricky to find, but the vscode test fixtures folder has a bunch more samples - https://github.com/dev-cycles/contextive/tree/main/src/vscode/contextive/test/single-root/fixtures/simple_workspace/.contextive

I might use one (or more) of those for some demo repos.
